cmd/server/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 	}
 
 	database.SetupDb(cfg.GetDatabaseConfig())
+	database.PopulateDb()
```

Comment:
hmm i dont think its a good idea to couple seeding with starting the server (itll run everytime the code redeploys/code changes for people using make dockerup, which uses air, which is kinda like nodemon)

could u make a `cmd/seeddb/main.go`?